packages/zwave-js/src/lib/commandclass/SceneControllerConfigurationCC.ts

```diff
@@ -0,0 +1,490 @@
+		return this.getValueDB().getValue(getGroupCountValueId()) ?? 0;
+	}
+}
+
+interface SceneControllerConfigurationCCSetOptions extends CCCommandOptions {
+	groupId: number;
+	sceneId: number;
+	dimmingDuration: Duration;
+}
+
+@CCCommand(SceneControllerConfigurationCommand.Set)
+export class SceneControllerConfigurationCCSet extends SceneControllerConfigurationCC {
+	public constructor(
+		driver: Driver,
+		options:
+			| CommandClassDeserializationOptions
+			| SceneControllerConfigurationCCSetOptions,
+	) {
+		super(driver, options);
+		if (gotDeserializationOptions(options)) {
+			// TODO: Deserialize payload
+			throw new ZWaveError(
+				`${this.constructor.name}: deserialization not implemented`,
+				ZWaveErrorCodes.Deserialization_NotImplemented,
+			);
+		} else {
+			const groupCount = this.getGroupCountCached();
+			this.groupId = options.groupId;
+			this.sceneId = options.sceneId;
+			this.dimmingDuration = options.dimmingDuration;
+
+			if (this.groupId < 2 || this.groupId > groupCount) {
+				throw new ZWaveError(
+					`${this.constructor.name}: The user ID must be between 2 and the number of supported groups ${groupCount}.`,
+					ZWaveErrorCodes.Argument_Invalid,
+				);
+			}
+		}
+	}
+
+	public groupId: number;
+
+	public sceneId: number;
+
+	public dimmingDuration: Duration;
+
+	public serialize(): Buffer {
+		this.payload = Buffer.from([
+			this.groupId,
+			this.sceneId,
+			this.dimmingDuration.serializeSet(),
+		]);
+		return super.serialize();
+	}
+
+	public toLogEntry(): MessageOrCCLogEntry {
+		return {
+			...super.toLogEntry(),
+			message: {
+				"group id": this.groupId,
+				"scene id": this.sceneId,
+				"dimming duration": this.dimmingDuration.toString(),
+			},
+		};
+	}
+}
+
+@CCCommand(SceneControllerConfigurationCommand.Report)
+export class SceneControllerConfigurationCCReport extends SceneControllerConfigurationCC {
+	public constructor(
+		driver: Driver,
+		options: CommandClassDeserializationOptions,
+	) {
+		super(driver, options);
+		validatePayload(this.payload.length >= 3);
+		this.groupId = this.payload[0];
+		this.sceneId = this.payload[1];
+		this.dimmingDuration =
+			Duration.parseReport(this.payload[2]) ?? new Duration(0, "unknown");
+
+		this.persistValues();
+	}
+
+	public readonly groupId: number;
+	public readonly sceneId: number;
+	public readonly dimmingDuration: Duration;
+
+	public persistValues(): boolean {
+		persistSceneConfig.call(
+			this,
+			this.groupId,
+			this.sceneId,
+			this.dimmingDuration,
+		);
+		return true;
+	}
+
+	public toLogEntry(): MessageOrCCLogEntry {
+		return {
+			...super.toLogEntry(),
+			message: {
+				"group id": this.groupId,
+				"scene id": this.sceneId,
+				"dimming duration": this.dimmingDuration.toString(),
+			},
+		};
+	}
+}
+
+interface SceneControllerConfigurationCCGetOptions extends CCCommandOptions {
+	groupId: number;
+}
+
+@CCCommand(SceneControllerConfigurationCommand.Get)
+@expectedCCResponse(SceneControllerConfigurationCCReport)
```

Comment:
This should have a predicate whether the correct report is received, see `testResponseForAlarmSensorGet` as an example. You can special-case the id 0 there to allow any report.